### ISSUE TITLE
Should enforce the framework style import

### ISSUE THREAD

**Issue by woshimaliang** [pinterest/PodToBUILD#109]
Today,  both import styles work when import framework headers:  

`#import "SomeFrameworkHeader.h"  `

`#import <SomeFramework/SomeFrameworkHeader.h>`

And ideally only the framework style import should be allowed.

**Comment by rahul-malik** [comment]
I agree this would be nice to enforce but flagging as an area of improvement for now since it isn't affecting builds
